test/configs/Util.ts

```diff
@@ -34,7 +34,7 @@ import {
   PutOperationHandler,
   RawBodyParser,
   RepresentationConvertingStore,
-  SequenceHandler,
+  SequenceHandler, SingleRootContainerManager,
```

Comment:
Multiline?

Reply:
I should find out if I can force my IDE to stop doing this.

Reply:
I had hoped for a rule in https://github.com/benmosher/eslint-plugin-import, but apparently there is none